drivers/sensor/current_amp/current_amp.c

```diff
@@ -8,27 +8,36 @@
 
 #include <zephyr/drivers/adc.h>
 #include <zephyr/drivers/adc/current_sense_amplifier.h>
+#include <zephyr/drivers/gpio.h>
 #include <zephyr/drivers/sensor.h>
+#include <zephyr/pm/device.h>
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(current_amp, CONFIG_SENSOR_LOG_LEVEL);
 
+struct current_amp_config {
+	struct current_sense_amplifier_dt_spec current;
```

Comment:
Not a block but for me when I read the code it will be more clear if I read `sensor` or `current_sensor`.

Reply:
`current_sense` ok?

Reply:
@nandojve with the [below](https://github.com/zephyrproject-rtos/zephyr/pull/62992#discussion_r1334961553) this field is now removed.